Sources/AngouriMath/Core/Entity/Continuous/Entity.Continuous.Complex.Definition.cs

```diff
@@ -36,6 +36,11 @@ namespace AngouriMath
                 /// Imaginary part of a complex number
                 /// </summary>
                 public Real ImaginaryPart => imaginary ?? Integer.Zero;
+                
+                /// <summary>
+                /// Conjugate of a complex number. Given this = a + ib, Conjugate = a - ib
+                /// </summary>
+                public Complex Conjugate => Create(RealPart, -ImaginaryPart);
```

Comment:
Great!
There's just one minor thing which I forgot to mention. Since it's a property, it should have a caching field. So something like
```cs
public Complex Conjugate => conjugate.GetValue(@this => @this.Create(@this.RealPart, -@this.ImaginaryPart), this);
private FieldCache<Complex> conjugate;
```
(so that we don't recreate it on every call)